PWG/EMCAL/EMCALtasks/AliAodSkimTask.cxx

```diff
@@ -157,7 +157,7 @@ void AliAodSkimTask::UserExec(Option_t *)
 
   // Accept event only if an track with a minimum pT of fTrackMinPt has been found in the event
   Bool_t storePt = kFALSE;
-  if (fTrackMinPt > 0 ){
+  if (fTrackMinPt > 0 && fTrackMaxPt < 0){
```

Comment:
Not sure about this upper check, should we not check the "or" if any of the two are turned on (-1 means not used)

Reply:
In principle the default situation is fTrackMaxPt=-1    so  if fTrackMaxPt is not set by user  and you give just fTrackMinPt > 0   the macro should behave as before,  you select all events with pT> fTrackMinPt .